### DISCUSSION TITLE
How ATIP is being used

### DISCUSSION THREAD

**Discussion by robinlovelace-ate** [acteng/atip#141] [General]
It's of interest to see how ATIP is being used in the real world. Feedback from colleagues in ATE shows how valuable the data is:

> The inspections team at Active Travel England used ATIP to quickly determine location, context and whether the correct approach was being taken with the proposed scheme assets. Furthermore, ATIP gave the team a chance to assess network cohesion across multiple government programmes and compliance with LCWIP (Local Cycling and Walking Infrastructure Plan) priorities. 

Any other ways that ATIP is or could be used welcome here!

**Comment by buildvoc10** [comment]
Hello using for walking routes, public footpaths and public roads

**Reply by robinlovelace-ate** [reply]
Great stuff thanks @buildvoc10 can you provide more context and details? Screenshots would be great to help guide development and inspire ATIP developers!

**Reply by buildvoc10** [reply]
@robinlovelace-ate  OK will upload screenshots
